packages/ui-tree-browser/src/TreeBrowser/TreeNode/styles.css

```diff
@@ -0,0 +1,328 @@
+/* stylelint-disable no-descending-specificity, selector-max-class */
+
+/**
+  TODO: this is a copy of the Button's css, keep in sync!
```

Comment:
I think you should add the sync warning to the Button's css file as well.

Reply:
done